code/modules/paperwork/paperplane.dm

```diff
@@ -0,0 +1,102 @@
+// Ported from TG. Known issue: Throw hit can possibly double-proc. Seems to be throw code.
+/obj/item/weapon/paperplane
+	name = "paper plane"
+	desc = "Paper, folded in the shape of a plane."
```

Comment:
`Paper folded into the shape of a plane.`

Sounds a bit better, but its up to you.